src/main/java/org/zone/commands/structure/zone/flags/greetings/GreetingsMessageViewCommand.java

```diff
@@ -0,0 +1,56 @@
+package org.zone.commands.structure.zone.flags.greetings;
+
+import net.kyori.adventure.text.Component;
+import org.spongepowered.api.command.CommandResult;
+import org.zone.commands.system.ArgumentCommand;
+import org.zone.commands.system.CommandArgument;
+import org.zone.commands.system.NotEnoughArgumentsException;
+import org.zone.commands.system.arguments.operation.ExactArgument;
+import org.zone.commands.system.arguments.zone.ZoneArgument;
+import org.zone.commands.system.context.CommandContext;
+import org.zone.region.Zone;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.greetings.GreetingsFlag;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class GreetingsMessageViewCommand implements ArgumentCommand {
+    public static final ExactArgument ZONE = new ExactArgument("zone");
+    public static final ExactArgument FLAGS = new ExactArgument("flags");
+    public static final ZoneArgument ZONE_VALUE = new ZoneArgument("zone_value", new ZoneArgument.ZoneArgumentPropertiesBuilder());
+    public static final ExactArgument GREETINGS = new ExactArgument("greetings");
+    public static final ExactArgument MESSAGE = new ExactArgument("message");
```

Comment:
You are not using MESSAGE. please remove this line

Reply:
Done will update